templates/Gemfile.erb

```diff
@@ -37,7 +37,8 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "pry-byebug"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.4.0"
+  gem "refills"
+  gem "rspec-rails", "~> 3.5.0.beta4"
```

Comment:
We would be losing these commits:

https://github.com/rspec/rspec-rails/commit/1fad32257f0970936f598b64de8ef4a40ede058d
https://github.com/rspec/rspec-rails/commit/5612e0d45ac4994fa72db41e611b4c6cbc2a33b6

It seems fine to do without those, and we can upgrade Suspenders next week after `rspec-rails` 3.5.0 is out.